src/js/api.js

```diff
@@ -97,32 +101,83 @@ export class API {
 
   getServiceRequests() {
     const url = `${this.baseUrl}/api/service-requests/`;
-    return $.get(url);
+    return $.get({
+      url,
+      headers: {
+        authorization: `Bearer ${getAccessToken()}`,
+      },
```

Comment:
Just a note that we might want to consider abstracting this in a function. Something like:

```
function getPayload(url) {
	return {
		url,
		headers: {
        authorization: `Bearer ${getAccessToken()}`,
      },
	}
}
```